test/scripts/angry-tock.js

```diff
@@ -421,15 +421,17 @@ describe('Angry Tock', () => {
       })
     ).to.equal(true);
 
+    console.log(robot.messageRoom.args[2]);
```

Comment:
Do you want to remove the console.log()? Does it matter?

Reply:
Oops